.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,27 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
```

Comment:
should we generate multiple versions or does the latest version is enough ?

Reply:
ping @jo-chemla

Reply:
The new version is probably enough. 

If everything is set-up correctly, every new github release of f3d should therefore automatically create a PR to winget-pkgs - or you can fire a winget-release by executing the action manually and specifying a version tag number. At least this is what is happening now to triplex for which I did the same PR, see [here](https://github.com/try-triplex/triplex/issues/218)